views/applications/all.html

```diff
@@ -30,9 +30,9 @@ <h1 class="introduction--h1">{{structure.name}}</h1>
     {% for app in applications %}
       <tr class="base--tr">
         <td data-th="Application" class="base--td">{{app.name}}</td>
-        <td data-th="Live" class="base--td">{% if app.endpoints.live %}<a href="/{{config.base}}/{{a.id}}#live" class="base--link"><span class="endpoint-{{app.endpoints.live|first}}">{{app.endpoints.live|first}}</span></a>{% endif %}</td>
-        <td data-th="Updated" class="base--td">{% if app.endpoints.updated %}<a href="/{{config.base}}/{{a.id}}#updated" class="base--link"><span class="endpoint-{{app.endpoints.updated|first}}">{{app.endpoints.updated|first}}</span></a>{% endif %}</td>
-        <td data-th="Sunset" class="base--td">{% if app.endpoints.sunset %}<a href="/{{config.base}}/{{a.id}}#sunset" class="base--link"><span class="endpoint-{{app.endpoints.sunset|first}}">{{app.endpoints.sunset|first}}</span></a>{% endif %}</td>
+        <td data-th="Live" class="base--td">{% if app.endpoints.live %}<a href="/{{config.base}}/{{a.id}}#live" class="base--link"><span class="endpoint-{{app.endpoints.live.response|first}}">{{app.endpoints.live.response|first}}</span></a>{% endif %}</td>
```

Comment:
I'm not sure if `app.endpoints.live.response|first` will actually get the first item in `app.endpoints.live` and then the `response` property, but we can deal with that when we've got actual data in there if it doesn't work